Python/optimizer_bytecodes.c

```diff
@@ -118,14 +121,17 @@ dummy_func(void) {
             OUT_OF_SPACE_IF_NULL(res = sym_new_const(ctx, temp));
```

Comment:
Don't you need to decref it here too on this error branch? Or am I remembering the ownership wrongly? Same for everything below.

Might be better to write it as:
```
res = sym_new_const(ctx, temp);
Py_DECREF(temp);
OUT_OF_SPACE_IF_NULL(res);
```

Reply:
Good catch!